src/Header/header.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import Navigation from '../Navigation/navigation';
+
+export default class Header extends React.Component{
+    constructor(){
+        super()
+        this.state = {
+            opener: true,
```

Comment:
This would be better as an enum. Currently as a reader of the code, it's hard to know what `true`, `false` and `null` mean.